mergify_engine/engine/commands_runner.py

```diff
@@ -37,7 +37,10 @@
 
 LOG = daiquiri.getLogger(__name__)
 
-COMMAND_MATCHER = re.compile(r"^@Mergify(?:|io) (\w*)(.*)", re.IGNORECASE)
+COMMAND_MATCHER = re.compile(
+    r"^(?:https://github.com/|@)Mergify(?:|io) (\w*)(.*)",
```

Comment:
We should use config.GITHUB_URL instead of the hardcoded URL to make it work with GHES

Reply:
Done @sileht